gen/definitions/snmp_version.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: SNMP Version
```

Comment:
This can be removed and added under the `SNMP VRF Host` resource as a nested list.